web-common/src/features/sources/add-source/FileDrop.svelte

```diff
@@ -53,14 +64,31 @@
           },
           "local_file"
         );
-        // TODO: errors
-        await createSource(
+        const errors = await createSource(
           queryClient,
           runtimeInstanceId,
           tableName,
           yaml,
           $createSourceMutation
         );
+
+        if (createSourceMutationError.isError || errors.length) {
```

Comment:
We should get the error for the source file path instead of assuming that the 1st error is for the source.